Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp

```diff
@@ -708,8 +710,25 @@ namespace AZ
 
         void DirectionalLightFeatureProcessor::CacheFullscreenPass()
         {
+#if USING_SCREENSHADOW_COMPUTEPASS
```

Comment:
Would it be possible to make this a runtime switch? would be nice to be able to compare quality and performance.